test.yaml

```diff
@@ -0,0 +1,32 @@
+name:Joplin Plugin
```

Comment:
This name and the file needs a relevant name. Also, the action should be under `.github/workflows` directory.